src/librustdoc/html/static/js/main.js

```diff
@@ -568,7 +568,11 @@ function preLoadCss(cssUrl) {
                 break;
             case "-":
                 ev.preventDefault();
-                collapseAllDocs();
+                collapseAllDocs(false);
+                break;
+            case "_":
```

Comment:
I don't think adding a new key to handle it is a good idea. What you suggested with shift+minus sounds like a better approach.

Reply:
doesn't shift + minus send `_` on a standard US keyboard? or is there something I don't know about JS events?